File: server/user-service/src/main/java/com/sudoo/userservice/controller/dto/UserDto.java
package com.sudoo.userservice.controller.dto;

import com.sudoo.userservice.repository.entitity.Gender;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserDto {
    private String userId;

    private String fullName;

    private String phone;

    private LocalDate dob;

    private String bio;

    private String avatar;

    private String cover;

    private AddressDto address;

    private Gender gender;
}

File: server/cart-service/src/main/kotlin/com/sudoo/cartservice/controller/dto/UserDto.kt
package com.sudoo.cartservice.controller.dto

import java.util.*


data class UserDto(
        val userId: String,
        val fullName: String,
        var phone: String,
        var bio: String,
        var gender: String,
        var avatar: String,
        var cover: String,
        var address: String,
        val dob: Date,
)